teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
```

Comment:
- [ ] For this inheritance to work you need to import the person class here, as you have done in the  `student.rb` class file. Kindly do that.